src/Components/Dropdowns/Family/DropdownMenu.tsx

```diff
@@ -79,6 +84,22 @@ export class DropdownMenu extends Component<Props> {
     )
   }
 
+  private get selectedPosition(): number {
+    const {scrollToSelected, children} = this.props
+
+    if (!children || !scrollToSelected) {
+      return 0
+    }
+
+    const itemHeight = 24
```

Comment:
This may not always be 24, but in most cases it will be. Probably not a huge deal if it ends up scrolling slightly off. I'd imagine detecting the height of children would be complex

Reply:
We could obtain the height of the child but we'll still run into issues if the children are different heights.

Reply:
Merging this as MVP. Will make component smarter to find the top position after #111 is complete.